test/k8sT/manifests/kafka-sw-security-policy.yaml

```diff
@@ -4,6 +4,17 @@ description: "Allow only permitted Kafka requests to empire Kafka broker"
 metadata:
   name: "secure-empire-kafka"
 specs:
+  - endpointSelector:
+      matchLabels:
+        app: kafka
+    ingress:
+    - fromEndpoints:
+      - matchLabels:
+          app: kafka
+      toPorts:
```

Comment:
Naive question:

I changed the GSG like so:

```  - endpointSelector:
      matchLabels:
        app: kafka
    ingress:
    - fromEndpoints:
      - matchLabels:
          app: empire-hq
      toPorts:
      - ports:
        - port: "9092"
          protocol: TCP
        rules:
          kafka:
          - apiKey: "apiversions"
          - apiKey: "metadata"
          - apiKey: "produce"
            topic: "deathstar-plans"
          - apiKey: "produce"
            topic: "empire-announce"
    - fromEndpoints:
      - matchLabels:
          app: kafka
```

My question : Is the `toPorts` needed in this case? or will the following code suffice:
```
  - fromEndpoints:
      - matchLabels:
          app: kafka
```
If not, we need to update the GSG guide as well.

Reply:
The toPorts is not necessary, it's added protection.

Reply:
I guess we should update GSG too in that case? I will take care of it! thanks